02.general/17.dns-providers/docs.en.md

```diff
@@ -428,3 +433,85 @@ Regular and DNS-over-TLS servers which provide protection from phishing and spyw
 | Protocol       | Address                                            |                |
 |----------------|----------------------------------------------------|----------------|
 | DNSCrypt, IPv4 | Provider: `2.dnscrypt-cert.nawala.id` IP: `180.131.144.144`  | <a href="sdns://AQAAAAAAAAAADzE4MC4xMzEuMTQ0LjE0NCDGC-b_38Dj4-ikI477AO1GXcLPfETOFpE36KZIHdOzLhkyLmRuc2NyeXB0LWNlcnQubmF3YWxhLmlk">Add to AdGuard</a> |
+
+<a name="securedns.eu-dns"></a>
+
+### securedns.eu
+
+[DoT, DoH, DNSCRYPT Provider securedns.eu](https://securedns.eu/) DoT, DoH, DNSCRYPT Provider securedns.eu
+
+| Protocol       | Address                                            |                |
+|----------------|----------------------------------------------------|----------------|
+| DoT, IPv4 | Provider: `dot.securedns.eu` IP: `146.185.167.43:853 ` Hostname: `dot.securedns.eu` | <a href="tls://securedns.eu">Add to AdGuard</a> |
+| DoT, IPv6 | Provider: `dot.securedns.eu` IP: `[2a03:b0c0:0:1010::e9a:3001]:853 ` Hostname: `dot.securedns.eu` | <a href="tls://securedns.eu">Add to AdGuard</a> |
```

Comment:
AdGuard for Android has a configured intent-filter for the sdns scheme. This is why you need to translate DoT(`tls://…`) and plain-dns into sdns-stamps according to https://dnscrypt.info/stamps-specifications (you can also use a simplified scheme without bootstrap-dns and hashes).
For the same reason, it is possible not to separate DoT ipv4 and ipv6

Reply:
i think i need some help with this :( for dnsc and doh it's clear (frank @jedisc1 made a online tool) but for tls and clean dns in sdns:// stamps i get made

Reply:
Hi!
Here is a simple example for DoT and plain DNS: https://play.golang.org/p/fACN5hsChEG

Reply:
cool stuff <3 thanks i will update maybe on sunday or monday evening, so this werkend i‘ve got a cultural and artevent in the alps... have to work for money

Reply:
@admitrevskiy 
do diem :)

thanks a lot to the golang playgroundlinks about tls in sdns:// stuff

i did some changes in the code [https://github.com/AdguardTeam/AdguardKnowledgeBase/pull/321/commits/b6d1c8cb7536f730b290d003e636e2dbf598a56b](https://github.com/AdguardTeam/AdguardKnowledgeBase/pull/321/commits/b6d1c8cb7536f730b290d003e636e2dbf598a56b) please review

have fun